storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -1037,7 +1038,8 @@ struct my_option xb_client_options[] = {
     {"no-backup-locks", OPT_NO_BACKUP_LOCKS,
      "This option controls if "
      "backup locks should be used instead of FLUSH TABLES WITH READ LOCK "
-     "on the backup stage. The option has no effect when backup locks are "
+     "on the backup stage. It will disable lock-ddl. The option has no effect "
+     "when backup locks are "
      "not supported by the server. This option is enabled by default, "
```

Comment:
> This option is enabled by default

Do we enable or disable by default?

Reply:
no

Reply:
This is not related to the change you are doing on this PR, but if you read the whole info about `--no-backup-locks` it says in the end that:
> " This option is enabled by default, "

However, if you look the default, it's off:
```
(uchar *)&opt_no_backup_locks, (uchar *)&opt_no_backup_locks, 0, GET_BOOL,
NO_ARG, 0, 0, 0, 0, 0, 0},
```

Description and default value are contradicting themselves.

Reply:
fixed the help message